src/js/structs/PodInstance.js

```diff
@@ -44,6 +51,18 @@ module.exports = class PodInstance extends Item {
     }
   }
 
+  getLastChanged() {
+    return new Date(this.get('lastChanged'));
+  }
+
+  getLastUpdated() {
+    return new Date(this.get('lastUpdated'));
+  }
+
+  getName() {
```

Comment:
Why this duplicate? This confuses me quite a bit.
I am going to remove it. If you need it back then implement it again and then let's take it from there.